gitgud/operations.py

```diff
@@ -155,6 +157,54 @@ def commit(self, commit_message, parents, time_offset):
                 skip_hooks=True)
         return commit_obj
 
+    def file_in_commit(self, commit, filepath):
```

Comment:
We should be consistent with accepting `Commit`s or `string`s. I think we should accept both in all cases